bin/epaa.py

```diff
@@ -679,12 +679,25 @@ def get_matrix_max_score(allele, length):
         return np.nan
 
 
+def create_score_values(j, method):
```

Comment:
As mentioned in the other comment it might be better to also support the case in which we use a tool other than `syfpeithi` that reports a score.